rating-service/src/main/kotlin/com/example/rating_service/RatingServiceApplication.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.rating_service
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+import org.springframework.context.annotation.ComponentScan
+
+@SpringBootApplication
+@ComponentScan(basePackages = ["com.example.rating_service"])
+open class RatingServiceApplication
```

Comment:
No need for open class

Reply:
The class needs to be open; otherwise, Spring throws an error saying the @Configuration class cannot be final